test/resnet/resnet_torchvision.py

```diff
@@ -0,0 +1,43 @@
+"""Test whether torchvision is extendable with graph utils."""
+from test.utils.skip_test import skip_pytorch_below_1_9_0
+from typing import Tuple
+
+import pytest
+import torchvision.models
+from torch import Tensor, rand, rand_like
+from torch.nn import Module, MSELoss
+
+from backpack import backpack, extend
+from backpack.extensions import DiagGGNExact
+
+
+@pytest.fixture
+def model_and_input() -> Tuple[Module, Tensor]:
+    """Yield ResNet model and an input to it.
+
+    Yields:
+        model and input
+    """
+    skip_pytorch_below_1_9_0()
+    resnet18 = torchvision.models.resnet18(num_classes=4).eval()
+    yield extend(resnet18, use_converter=True), rand(2, 3, 7, 7, requires_grad=True)
+    del resnet18
+
+
+def test_network_diag_ggn(model_and_input):
+    """Test whether the given module can compute diag_ggn.
+
+    Args:
+        model_and_input: module to test
+    """
+    model, x = model_and_input
+
+    result = model(x)
+    loss = extend(MSELoss())(result, rand_like(result))
+
+    with backpack(DiagGGNExact()):
+        loss.backward()
+    for name, param in model.named_parameters():
+        print(name)
+        print(param.grad.shape)
+        print(param.diag_ggn_exact.shape)
```

Comment:
There is no `assert` in this test. Is it an integration test?

Reply:
I'm not sure what an integration test is. This test is intended to just check whether resnet18 runs with BackPACK. It is placed seperately instead of diag_ggn_settings, because PyTorch is not able to compute on resnet18.

Meanwhile I added an assert to compare the result to the not converted version.

Additionally, now, I improved the docstring to clarify why this test is there and what it does.